### PR TITLE
Redesign Exception Template

### PR DESCRIPTION
**Pull request by reibengu**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #2

### CODE REVIEW

File: templates/exception.php [hunk 1]
```diff
@@ -9,515 +9,694 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8"/>
-    <title><?= $this->htmlEncode($this->getThrowableName($throwable)) ?></title>
+    <meta charset="utf-8" />
+    <title>
+        <?= $this->htmlEncode($this->getThrowableName($throwable)) ?>
+    </title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700">
```
Comment (samdark) [inline templates/exception.php:16]: I'd use local font instead. Yii may be used in local environments w/o Internet connection.

File: templates/exception.php [hunk 2]
```diff
@@ -9,515 +9,694 @@
...
+            position: absolute;
+            z-index: 200;
+            display: block;
+            width: 25px;
+            text-align: right;
+            color: #aaa;
+            line-height: 20px;
+            font-size: 12px;
+            margin-top: 1px;
+            font-family: Consolas, monospace;
```
Comment (samdark) [inline templates/exception.php:275]: I'd add https://www.jetbrains.com/lp/mono/ to the list.

File: templates/exception.php [hunk 3]
```diff
@@ -9,515 +9,694 @@
...
-    </div>
+        <div class="footer">
+            <div class="flex-1">
+                <p class="timestamp">
+                    <?= date('Y-m-d, H:i:s') ?>
+                </p>
+                <p>
+                    <?= $this->createServerInformationLink() ?>
+                </p>
+                <p><a href="http://yiiframework.com/">Yii Framework</a>/
```
Comment (samdark) [inline templates/exception.php:466]:
```suggestion
                <p><a href="https://www.yiiframework.com/">Yii Framework</a>/
```

File: templates/previousException.php [hunk 4]
```diff
@@ -4,11 +4,13 @@
...
-    <h3><?= nl2br($this->htmlEncode($throwable->getMessage())) ?></h3>
-    <p>in <span class="file"><?= $throwable->getFile() ?></span> at line <span class="line"><?= $throwable->getLine() ?></span></p>
-    <?= $this->renderPreviousExceptions($throwable) ?>
+    <div class="flex-1">
+        <h2>
+            <span>Caused by:</span>
+            <span><?= $this->htmlEncode(get_class($throwable)) ?></span>
+        </h2>
+        <h3><?= nl2br($this->htmlEncode($throwable->getMessage())) ?></h3>
+        <p>in <span class="file"><?= $throwable->getFile() ?></span> at line <span class="line"><?= $throwable->getLine() ?></span></p>
```
Comment (samdark) [inline templates/previousException.php:13]:
```suggestion
        <p>in <span class="file"><?= $this->htmlEncode($throwable->getFile()) ?></span> at line <span class="line"><?= $throwable->getLine() ?></span></p>
```

File: templates/exception.php (hunk 1 above)
Comment (reibengu) [reply templates/exception.php:16]: How can we use this font locally if we don't include it in public/assets or public/vendor/assets?

Comment (samdark) [reply templates/exception.php:16]: You're right. That would complicate things a bit too much.

**Comment by reibengu** [comment]
@samdark should be ready for testing.

**Review by samdark — APPROVED** [review]
Looks great now!

**Comment by samdark** [comment]
Thank you!

**Pull request merged**